### PR TITLE
Body and file schema

### PR DESCRIPTION
**Pull request by isaacna**
**Pull request recommendations:**
- [x] Name your pull request _your-development-type/short-description_. Ex: _feature/read-tiff-files_
- [x] Provide context of changes.
* I didn't really see anything about `file` that should be different from the existing schema.
* For body, I added `external_source_id`, `ended`, `chair_person_id`, and `is_select`. I thought including the chair would be useful for quickly finding the active chair for a given body. 
* I also thought adding a flag for whether a body is select or not wouldn't hurt, since it doesn't add any complexity but could be a useful bit of info.

### CODE REVIEW

File: database_schema.md [hunk 1]
```diff
@@ -75,6 +75,29 @@ person_id: {
 }
 ```
 
+### Body
+```
+body_id: {
+    name: str
+    description: str
+    created: datetime
+    ended: datetime
```
Comment (tohuynh) [inline database_schema.md]: What would be `created` and `ended` datetime values for City Council and Council Briefing bodies?
And how would we query to get all the active bodies?

File: database_schema.md [hunk 2]
```diff
@@ -75,6 +75,29 @@ person_id: {
...
+    chair_person_id: str
+    external_source_id: str
+    is_select: bool
```
Comment (tohuynh) [inline database_schema.md]: Is `is_select` meant to capture whether the body is something like 'Select Committee on Campaign Reform'? That's fine if that is the case.

File: database_schema.md [hunk 3]
```diff
@@ -75,6 +75,29 @@ person_id: {
...
+}
+```
+
+### File
+```
+file_id: {
```
Comment (tohuynh) [inline database_schema.md]: I agree with the design for File collection.

File: database_schema.md (hunk 2 above)
Comment (isaacna) [reply database_schema.md]: Yeah it was supposed to be a flag for whether a committee was "Select" vs. "Standing" like [here](https://www.seattle.gov/council/committees). But could generalize this to `tag: str` for versatility in filtering if it's ever needed.

File: database_schema.md (hunk 1 above)
Comment (isaacna) [reply database_schema.md]: I was thinking that any body that is permanently ongoing can have be null/empty for those fields. And we could query for whether `ended` is present if we want the active bodies.

If storing null/empty values isn't ideal or best practice we can switch to `is_active: bool` instead.

Comment (tohuynh) [reply database_schema.md]: Querying for null is kind of icky in Firebase. Plus we would have to query with the condition: `ended == null or today's date < ended`, to get all the active bodies.

With `is_active`, we need only one condition in query

Comment (JacksonMaxfield) [reply database_schema.md]: I like `is_active` a bit more than just having `ended`. Having all three, while duplicated data, is "simpler" to understand.

File: database_schema.md (hunk 2 above)
Comment (JacksonMaxfield) [reply database_schema.md]: Let's go with `tag`

File: database_schema.md (hunk 1 above)
Comment (tohuynh) [reply database_schema.md]: I'm OK with having all three fields.

**Pull request merged**